include/tlog/errs.h

```diff
@@ -141,4 +141,66 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to the "perrs" stack, with message as a constant string
+ * and then go to cleanup.
+ *
+ * @param _msg   The message string to copy and put into the pushed error.
+ *               Cannot be NULL.
+ */
+#define TLOG_ERRS_RAISES(_msg) \
+     do {                                                                 \
+         tlog_errs_pushs(perrs, _msg);                                    \
+         goto cleanup;                                                    \
+     } while (0)
+
+/**
+ * Push two errors to the "perrs" stack, first one with message being a global
+ * return code description, and the second with message as a constant string.
+ * Assign _grc argument to "grc", and then go to cleanup.
+ *
+ * @param grc   The global return code, which description will be pushed to
+ *              the stack.
+ * @param _msg  The message string to copy and put into the pushed error.
+ *              Cannot be NULL.
+ */
+#define TLOG_ERRS_RAISECS(_grc, _msg) \
+     do {                                                                 \
+         grc = _grc;                                                      \
+         tlog_errs_pushc(perrs, grc);                                     \
+         tlog_errs_pushs(perrs, _msg);                                    \
+         goto cleanup;                                                    \
+     } while (0)
+
+/**
+ * Push an error to the "perrs" stack, with a sprintf-formatted message.
+ * and then go to cleanup.
+ *
+ * @param fmt   The message sprintf(3) format string. Cannot be NULL.
```

Comment:
Please make the parameter name here match the definition. I.e. prepend `fmt` with `_`.